pharmd-app/src/components/Fields/ChipField.js

```diff
@@ -2,28 +2,25 @@ import React from "react";
 import tw, { styled } from "twin.macro";
 import Chip from "@material-ui/core/Chip";
 
-// ${props => props.theme.typography.size4}
 const Pill = styled(Chip)`
 	
    ${tw`rounded-lg capitalize w-28 fontStyle-4 font-bold tracking-wider`}
-	
+   font-size: ${props => props.size == "small" ? `.9em` : `16px`};
```

Comment:
I wonder if size should be a boolean since there's only two possibilities?

Reply:
i was considering of that option when setting up this conditional, but I'm not sure if it's possible since `size` is passed from `Pill` where the size is either `small` or `medium` as far as i know